src/frontend/get-github-setup.ts

```diff
@@ -8,16 +8,10 @@ Redirects users back to github/configuration to install their Jira instance in G
 If the installation was done from Jira Marketplace, the app is already installed.
 */
 export default (req: Request, res: Response, next: NextFunction): void => {
-
 	req.log.info("Received get github setup page request");
-
-	if (req.session.jiraHost) {
-		const { host: githubHost, session } = req;
-		const { jwt, jiraHost } = session;
-
-		const urlArgs = { githubHost, jwt, jiraHost };
-
-		return res.redirect(getGitHubConfigurationUrl(urlArgs));
+	const { host: githubHost, session: { jwt, jiraHost } } = req;
+	if (jiraHost && jwt) {
```

Comment:
What's the impact of adding `&& jwt` condition here?

Reply:
None as we need both to continue.  I figured might as well check for jwt as well since the typings were complaining that jwt might be undefined.